Uploadable/MimeTypeGuesserAdapter.php

```diff
@@ -2,14 +2,15 @@
 
 namespace Stof\DoctrineExtensionsBundle\Uploadable;
 
-use Symfony\Component\HttpFoundation\File\MimeType\MimeTypeGuesser;
+use Symfony\Component\Mime\MimeTypes;
 use Gedmo\Uploadable\MimeType\MimeTypeGuesserInterface;
 
 class MimeTypeGuesserAdapter implements MimeTypeGuesserInterface
 {
     public function guess($filePath)
     {
-        return MimeTypeGuesser::getInstance()->guess($filePath);
+        $mimeTypes = new MimeTypes();
```

Comment:
MimeTypes has a heavy constructor. Perhaps you could call `MimeTypes::getDefault()` or add MimeTypes as a constructor argument to have it injected.

Reply:
Thank you for your comment.
Changed to the static getDefault() Variant. Dont know how to do it with constructor, i assume i would have to configure this as a service? 
Are there relevant advantages?

Reply:
I don't think there are important advantages. Maybe someone configures something specific in their application. But if that is the case, they should open a PR :)